[WKCI] Add Tahoe-eligible bots to post-commit Sequoia x86_64 release/debug testing queues.

bls1999 opened the pull request:
#### f934b9407ea513e0ec68c3cf902223cf2cf932ce
<pre>
[WKCI] Add Tahoe-eligible bots to post-commit Sequoia x86_64 release/debug testing queues.
<a href="https://bugs.webkit.org/show_bug.cgi?id=298371">https://bugs.webkit.org/show_bug.cgi?id=298371</a>
<a href="https://rdar.apple.com/159823842">rdar://159823842</a>

Reviewed by Ryan Haddad.

As Macmini8,1 won&apos;t be able to update to Tahoe, this change adds x86_64
hardware that will be able to update to Tahoe (MacPro7,1) to some queues
currently only served by Macmini8,1s.

* Tools/CISupport/build-webkit-org/config.json:

Canonical link: <a href="https://commits.webkit.org/299579@main">https://commits.webkit.org/299579@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/5bf4b1bbbeecd990c2e79e264fe1f48c2da8d4e4

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/119372 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/39059 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/29714 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/125620 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/71443 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/1a3d2fd5-e0c8-4489-8663-5782ce700ac2) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/121249 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/39756 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/47640 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/90701 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/60021 "Passed tests") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/866aaba7-d959-459c-8e74-eef4ebb180da) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/122324 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/31721 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/107025 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/71152 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/6ff56e3a-65d1-45a7-bcc4-eebc689b16a9) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/30756 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/25134 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/69272 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/101175 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/25325 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/128617 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/46290 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/35027 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/99276 "Passed tests") | | 
| [✅ 🧪 services](https://ews-build.webkit.org/#/builders/28/builds/118795 "Passed tests") | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/46655 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/103224 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/99065 "Passed tests") | | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/44516 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/22519 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/42857 "Built successfully") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/19007 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/46153 "Built successfully") | | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/45618 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/48968 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/47305 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/5bf4b1bbbeecd990c2e79e264fe1f48c2da8d4e4. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50287

bls1999 committed:
[WKCI] Add Tahoe-eligible bots to post-commit Sequoia x86_64 release/debug testing queues.
https://bugs.webkit.org/show_bug.cgi?id=298371
rdar://159823842

Reviewed by Ryan Haddad.

As Macmini8,1 won't be able to update to Tahoe, this change adds x86_64
hardware that will be able to update to Tahoe (MacPro7,1) to some queues
currently only served by Macmini8,1s.

* Tools/CISupport/build-webkit-org/config.json:

Canonical link: https://commits.webkit.org/299579@main

webkit-commit-queue commented:
Committed 299579@main (f934b9407ea5): <https://commits.webkit.org/299579@main>

Reviewed commits have been landed. Closing PR #50287 and removing active labels.
